docs/meshstack.billing-configuration.md

```diff
@@ -3,11 +3,23 @@ id: meshstack.billing-configuration
 title: Configuration
 ---
 
-## Supported Currencies
+## Currency Conversion
 
-In order to simplify multi-cloud governance, cloud foundation teams should prefer standardising on a single chargeback currency.
-However, Cloud Foundation teams sometimes encounter situations in which they need to process billing or chargeback in multiple
-currencies. For example GCP might charge your organization for cloud consumption in EUR while Azure charges consumption in USD.
+To streamline cloud governance, meshStack recommends using a single chargeback currency. This functionality automatically converts $ from cloud providers like AWS into Euro. This simplification aids in the chargeback process to teams. 
+
+You have two options for enabling currency conversion in chargeback and usage reports:
+
+- Standard Currency Rate: By default, meshStack uses the conversion rates from frankfurter.app, which provides exchange rates from the European Central Bank. To activate this, navigate to the Financial Tab on the Settings page and enable the conversion feature using the provided toggle.
+- Customized Currency Rate: For a customized approach, you can supply your own conversion rates via API. First, deactivate the standard rate toggle in the Financial Tab. Then, contact our support team at support@meshcloud.io for assistance in integrating your system.
+
+If your meshStack instance is in a private environment with permissions for external requests, enable the default converter through the Settings page as described above.
+For environments with restricted external access, allow meshStack to communicate with external services by configuring your firewall rules accordingly.
+
+> Activating currency conversion affects both current and future months. It does not retroactively alter past cost data.
+
+### Other Supported Currencies 
+
+Sometimes Cloud Foundation teams encounter situations in which they need to process billing or chargeback in multiple currencies. For example GCP might charge your organization for cloud consumption in EUR while Azure charges consumption in USD.
```

Comment:
Does this mean the "base" currency can also be $? Is this already working?

Reply:
It is not a base but we are able to just show different currencies in one report.